classes/module/Module.php

```diff
@@ -2495,13 +2498,20 @@ public function fetch($templatePath, $cache_id = null, $compile_id = null)
      */
     protected function getCurrentSubTemplate($template, $cache_id = null, $compile_id = null)
     {
+        if ($compile_id === null) {
+            $compile_id = Context::getContext()->shop->theme->getName();
```

Comment:
why not introducing

    public function getDefaultCompileId()
    {
        return Context::getContext()->shop->theme->getName();
    }

?

Reply:
good point, fixed in a new commit